### PR TITLE
Adds creator user to event and shows that user and icon and link to profile in event show page

### LINKED ISSUE #1002: event show page should show who created the event

### PR DESCRIPTION
**Pull request by tansaku**
fixes #1002

### CODE REVIEW

**Comment by tansaku** [comment]
makes things look like:

**Comment by diraulo** [comment]
@tansaku we have some merge conflicts here

**Comment by tansaku** [comment]
thanks @diraulo should be fixed now

File: features/events/show_event.feature [hunk 1]
```diff
@@ -14,6 +17,15 @@ Feature: Show Events
 
 
   @javascript
+<<<<<<< HEAD
```
Comment (diraulo) [inline features/events/show_event.feature]: @tansaku this shouldn't be here 😉

**Comment by tansaku** [comment]
@diraulo doh!

**Pull request merged**